src/components/common/QuestionWrapper.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <section class="mt-5">
+    <header>
+      <h2 :id="id">{{ title }}</h2>
+      <p v-if="subtitle" class="pt-2">{{ subtitle }}</p>
+      <p v-if="comment" class="black--text mb-0">{{ comment }}</p>
+    </header>
+    <slot></slot>
+  </section>
+</template>
+
+<script>
+export default {
+  name: 'QuestionWrapper',
+  props: {
+    id: {
+      type: String,
+      default: ''
```

Comment:
If this is the id of this component then I recommend putting it on the top-most rendered element (line 2).